Resources/Prototypes/_Goobstation/Entities/Objects/Devices/Electronics/misc_signallers.yml

```
@@ -0,0 +1,71 @@
+      Plastic: 50
+      Glass: 25
+      Silver: 25
+
+# proximity signaller 
+- type: entity
+  parent: MiscSignallerBase
+  id: ProximitySignaller
+  name: proximity signaller
+  description: A device that transmits a signal when something is nearby. You may be able not to trigger it if you move slowly enough. Needs to be anchored.
+  components:
+    - type: Sprite
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)